app/shared/reducers/account/actions.js

```diff
@@ -0,0 +1,38 @@
+import * as types from '../../actions/types';
+
+const initialState = {
+  lastError: {},
+  lastTransaction: {},
+  list: [],
+  updated: null
+};
+
+export default function actions(state = initialState, action) {
+  switch (action.type) {
+    case types.RESET_ALL_STATES: {
+      return Object.assign({}, initialState);
+    }
+    case types.CLEAR_ACCOUNT_ACTIONS_CACHE: {
+      return Object.assign({}, state, {
+        list: []
+      });
+    }
+    case types.GET_ACCOUNT_ACTIONS_FAILURE: {
```

Comment:
This should only unset the account that failed after it's split to support multiple accounts.

Reply:
sounds good!